airbyte-integrations/connectors/source-python-http-tutorial/source_python_http_tutorial/source.py

```diff
@@ -38,8 +40,8 @@ def request_params(
         stream_slice: Mapping[str, Any] = None,
         next_page_token: Mapping[str, Any] = None,
     ) -> MutableMapping[str, Any]:
-        # The api requires that we include the base currency as a query param so we do that in this method
-        return {"base": self.base}
+        # The api requires that we include access_key as a query param so we do that in this method ADD THIS
```

Comment:
the `ADD THIS`. comment is a little confusing. what needs to be added?

Reply:
nice catch! I forgot to delete that as i was marking what code to add to the tutorial. will delete this